# file: C:\Users\ADITI\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\scipy\interpolate\_fitpack_impl.py
# hypothesis_version: 6.135.14

[0.0, 1e-16, 'An error occurred', 'Error on input data', 'Invalid input data', 'Unknown error', 'bisplev', 'bisplrep', 'fp', 'ier', 'insert', 'iwrk', 'm > k must hold', 'raise', 'spalde', 'splantider', 'splder', 'splev', 'splint', 'splprep', 'splrep', 'sproot', 't', 'tx', 'ty', 'u', 'ub', 'ue', 'unknown', 'wrk']